bezier-rs/lib/src/lib.rs

```diff
@@ -453,6 +453,92 @@ impl Bezier {
 			.try_into()
 			.unwrap()
 	}
+
+	/// Determine if it is possible to scale the given curve, using the following conditions:
+	/// 1. all the control points are located on a single side of the curve.
+	/// 2. The on-curve point for t = 0.5 occurs roughly in the center of the polygon defined by the curve coordinates.
+	/// See [the offset section](https://pomax.github.io/bezierinfo/#offsetting) of Pomax's bezier curve primer for more details.
+	fn is_scalable(&self) -> bool {
+		// Verify all the control points are located on a single side of the curve.
+		if let BezierHandles::Cubic { handle_start, handle_end } = self.handles {
+			let angle_1 = (self.end - self.start).angle_between(handle_start - self.start);
+			let angle_2 = (self.end - self.start).angle_between(handle_end - self.start);
+			if (angle_1 > 0. && angle_2 < 0.) || (angle_1 < 0. && angle_2 > 0.) {
+				return false;
+			}
+		}
+		// Verify the on-curve point for t = 0.5 occurs roughly in the center of the polygon.
+		let normal_0 = self.normal(0.);
+		let normal_1 = self.normal(1.);
+		let s = normal_0.x * normal_1.x + normal_0.y * normal_1.y;
+		s.acos() < std::f64::consts::PI / 3.
```

Comment:
how did you pick this threshold? maybe make it evident that it is a constant (i.e. put it into a constant) so that future developers can change the threshold if needed.